packages/material-ui/src/Typography/Typography.js

```diff
@@ -108,8 +108,8 @@ const useTypographyClasses = (props) => {
   return utilityClasses;
 };
 
-const Typography = React.forwardRef(function Typography(inputProps, ref) {
-  const props = useThemeProps({ props: inputProps, name: 'MuiTypography' });
+const Typography = React.forwardRef(function Typography(inProps, ref) {
```

Comment:
I believe if the name is an issue we should update the other components to `inProps` too.